### PR TITLE
Add AppGate-Client 11.2.3

### PR DESCRIPTION
**Pull request by xavador**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/AppGate-Client.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'AppGate-Client' do
+  version '11.2.3'
+  sha256 :no_check
+
+  url "http://download.cryptzone.com/files/download/AppGate-#{version}/Clients/MacOSX/AppGate_Client.dmg"
+  name 'AppGate-Client'
+  homepage 'https://www.cryptzone.com/downloadcenter/appgate-classic/'
+
+  app 'AppGate Client 11.2.3.app'
```
Comment (vitorgalvao) [inline Casks/AppGate-Client.rb]: There is no such app inside the `dmg`. You need to interact with the `pkg` (documentation).

Also, it’s missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md) geared for that.

File: Casks/AppGate-Client.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'AppGate-Client' do
+  version '11.2.3'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/AppGate-Client.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/AppGate-Client.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  url "http://download.cryptzone.com/files/download/AppGate-#{version}/Clients/MacOSX/AppGate_Client.dmg"
+  name 'AppGate-Client'
+  homepage 'https://www.cryptzone.com/downloadcenter/appgate-classic/'
+
+  app 'AppGate Client 11.2.3.app'
+
+  caveats do
+    depends_on_java('6')
```
Comment (vitorgalvao) [inline Casks/AppGate-Client.rb]: Does it depend on this specific version, or is this the minimum?

File: Casks/AppGate-Client.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+cask 'AppGate-Client' do
+  version '11.2.3'
+  sha256 :no_check
+
+  url "http://download.cryptzone.com/files/download/AppGate-#{version}/Clients/MacOSX/AppGate_Client.dmg"
+  name 'AppGate-Client'
```
Comment (vitorgalvao) [inline Casks/AppGate-Client.rb]: `AppGate (classic)`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/AppGate-Client.rb (hunk 3 above)
Comment (xavador) [inline Casks/AppGate-Client.rb]: it is the minimum, should i change them? when yes, to what?

File: Casks/AppGate-Client.rb (hunk 2 above)
Comment (xavador) [inline Casks/AppGate-Client.rb]: thanks, i will change it with the next commit

File: Casks/AppGate-Client.rb (hunk 4 above)
Comment (xavador) [inline Casks/AppGate-Client.rb]: Should the name 'AppGate (classic)' and/or the complete cask?

File: Casks/AppGate-Client.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/AppGate-Client.rb]: To `6+`.

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Verifying locally, since Travis is taking too long.

**Pull request merged**